android/library/maply/src/main/java/com/mousebird/maply/BaseController.java

```diff
@@ -225,6 +225,9 @@ public void addMainThreadTaskAfter(double when,Runnable run) {
 	 */
 	public LayerThread getWorkingThread()
 	{
+		if (workerThreads == null)
+			return null;
+
```

Comment:
Seems like it can go null after being checked here?  Other access to `workerThreads` is synchronized.